.github/workflows/Dockerfile

```diff
@@ -0,0 +1,6 @@
+from espressopp/buildenv:ubuntu
+
+COPY espressopp/ /home/espressopp/espressopp
+RUN cmake -S espressopp -B espressopp-build -DCMAKE_BUILD_TYPE=Release -DEXTERNAL_MPI4PY=OFF
```

Comment:
Does the MPI4PY in the ubuntu image not work?

Reply:
I do not even know why you bundle mpi4py. @stuehn mentioned some hacky conversion between mpi4py communicator and c++ communicator. Perhaps a fixed version is needed for this conversion to work reliably.